src/middleware/streamingRouter.js

```diff
@@ -266,3 +266,29 @@ export function collectStream (readableStream) {
       })
   })
 }
+
+export function storeBodyInGridFs (body) {
```

Comment:
This function seems to be duplicated and has been implemented at other places. Lets see if we cant re-use the function for storing data into GridFS instead of creating another function doing the same thing.